Dockerfile

```diff
@@ -27,55 +27,60 @@
 # =================================================================
 #
 # Adapted from pycsw official Dockerfile
-# Contributors: Massimo Di Stefano
-# 
 #
-FROM alpine:latest
+# Contributors:
+#   Massimo Di Stefano
+#   Arnulf Heimsbakk
+#
+# Note in the instruction above I added some extra packages:
+# git - to clone pycsw repo or install with pip directly form github
+# bash - for interactive docker session
+# postgresql-dev, psycopg2 - for postgres backend
+# parmap - to easy parallelize routines like mmd2iso
+# xmltodict - pythonic way to work on xml
 
+FROM alpine:3.11
+MAINTAINER Senda <senda@met.no>
```

Comment:
s-enda is a project which will cease to exist at some point, is there a better candidate for a maintainer?

Reply:
Good point. I can set me or @epifanio  as maintainer.

Reply:
Fixed updated with adding two maintainers as LABEL, since MAINTAINER is deprecated by Docker.